code/game/machinery/iv_drip.dm

```diff
@@ -9,6 +9,19 @@
 	mouse_drag_pointer = MOUSE_ACTIVE_POINTER
 	var/obj/item/reagent_containers/iv_bag/bag = null
 
+/obj/machinery/iv_drip/Initialize(mapload)
+	. = ..()
+	if(mapload)
+		if(bag) // just in case anyone in the future ever adds IV drip subtypes with bags attached
+			return
+		for(var/obj/item/I in loc)
+			if(istype(I, /obj/item/reagent_containers/iv_bag))
+				I.forceMove(src)
+				bag = I
+				START_PROCESSING(SSmachines, src)
+				break
```

Comment:
```suggestion
		var/obj/item/reagent_containers/iv_bag/IV = locate(/obj/item/reagent_containers/iv_bag) in loc
		if(IV)
			IV.forceMove(src)
			bag = IV
			START_PROCESSING(SSmachines, src)
```
This should also work.
If the `update_icon` call needs to be added, then you can make this into an early return and return when `IV` is not found